tools/downloader/README.md

```diff
@@ -68,6 +68,108 @@ The cache format is intended to remain compatible in future Open Model Zoo
 versions, so you can use a cache to avoid redownloading most files when updating
 Open Model Zoo.
 
+Run the script with the --print_all option to see the available topologies:
+```sh
+./downloader.py --print_all
+Sphereface
+action-recognition-0001-decoder
+action-recognition-0001-encoder
+age-gender-recognition-retail-0013
+alexnet
+brain-tumor-segmentation-0001
+ctpn
+deeplabv3
+densenet-121
+densenet-121-tf
+densenet-161
+densenet-161-tf
+densenet-169
+densenet-169-tf
+densenet-201
+driver-action-recognition-adas-0002-decoder
+driver-action-recognition-adas-0002-encoder
+emotions-recognition-retail-0003
+face-detection-adas-0001
+face-detection-adas-binary-0001
+face-detection-retail-0004
+face-detection-retail-0044
+face-person-detection-retail-0002
+face-reidentification-retail-0095
+facenet-20180408-102900
+facial-landmarks-35-adas-0002
+faster_rcnn_inception_resnet_v2_atrous_coco
+faster_rcnn_inception_v2_coco
+faster_rcnn_resnet101_coco
+faster_rcnn_resnet50_coco
+gaze-estimation-adas-0002
+googlenet-v1
+googlenet-v2
+googlenet-v3
+googlenet-v4
+handwritten-score-recognition-0001
+head-pose-estimation-adas-0001
+human-pose-estimation-0001
+inception-resnet-v2
+inception-resnet-v2-tf
+instance-segmentation-security-0033
+instance-segmentation-security-0049
+landmarks-regression-retail-0009
+license-plate-recognition-barrier-0001
+license-plate-recognition-barrier-0007
+mobilenet-ssd
+mobilenet-v1-0.25-128
+mobilenet-v1-0.50-160
+mobilenet-v1-0.50-224
+mobilenet-v1-1.0-224
+mobilenet-v1-1.0-224-tf
+mobilenet-v2
+mobilenet-v2-1.4-224
+mtcnn-o
+mtcnn-p
+mtcnn-r
+pedestrian-and-vehicle-detector-adas-0001
+pedestrian-detection-adas-0002
+pedestrian-detection-adas-binary-0001
+person-attributes-recognition-crossroad-0230
+person-detection-action-recognition-0005
+person-detection-action-recognition-teacher-0002
+person-detection-raisinghand-recognition-0001
+person-detection-retail-0002
+person-detection-retail-0013
+person-reidentification-retail-0031
+person-reidentification-retail-0076
+person-reidentification-retail-0079
+person-vehicle-bike-detection-crossroad-0078
+resnet-101
+resnet-152
+resnet-50
+resnet50-binary-0001
+road-segmentation-adas-0001
+se-inception
+se-resnet-101
+se-resnet-152
+se-resnet-50
+se-resnext-101
+se-resnext-50
+semantic-segmentation-adas-0001
+single-image-super-resolution-1032
+single-image-super-resolution-1033
+squeezenet1.0
+squeezenet1.1
+ssd300
+ssd512
+ssd_mobilenet_v1_coco
+ssd_mobilenet_v2_coco
+text-detection-0002
+text-recognition-0012
+vehicle-attributes-recognition-barrier-0039
+vehicle-detection-adas-0002
+vehicle-detection-adas-binary-0001
+vehicle-license-plate-detection-barrier-0106
+vgg16
+vgg19
+```
```

Comment:
This isn't quite what I meant. There already is a description of `--print_all` down below; I just want you to update the example output in that description.

Reply:
@IRDonch I had made the update for the example output for existing ./TOOL.py --print_all. Is that what your looking for?

Reply:
Almost, but I don't think the _full_ output needs to be listed. It unnecessarily bloats the document and creates a maintenance burden. Please abbreviate it the way it's abbreviated now.

Reply:
Updated and it was abbreviated as before.